src/index.html

```diff
@@ -17,6 +17,7 @@
     <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.7.3/leaflet.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Gallery</title>
+    <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/scseqm3j';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()</script>
```

Comment:
We actually do want this in the head. It checks if `window.Intercom` has been created. If not, creates a `<script>` tag in the body to download intercom.js (the JS file download is the main thing we want in the body) and injects a placeholder that queues up `window.Intercom(...)` calls